test/albumController.test.js

```diff
@@ -173,24 +159,130 @@ describe('AlbumController API Tests', () => {
 
     await Album.insertMany(albumsToCreate);
 
-    // Make a GET request to the /api/v1/albums/filter endpoint with a limit parameter set to 4
     const limit = 3;
     const response = await request(app).get(
       `/api/v1/albums/filter?limit=${limit}`
     );
+    expect(response.status).toBe(StatusCodes.BAD_REQUEST);
 
-    // Assert that the response status is OK (200)
-    expect(response.status).toBe(StatusCodes.OK);
 
-    // Assert that the response body is an object with "albums" and "count" keys
+    expect(response.status).toBe(StatusCodes.OK);
     expect(response.body).toHaveProperty('albums');
     expect(response.body).toHaveProperty('count');
-
-    // Assert that "albums" is an array with a length equal to the specified limit
     expect(response.body.albums).toHaveLength(limit);
+    expect(response.body.count).toBe(albumsToCreate.length);
+  });
 
-    // Assert that the "count" field reflects the actual count of albums in the database
+  it('should create a new album - Success Case', async () => {
 
-    expect(response.body.count).toBe(albumsToCreate.length);
+    const user = await authenticateUser(User); 
+  
+      const newAlbumData = {
+        albumName: 'New Album',
+        artistName: 'New Artist',
+        price: 9.99,
+        spotifyUrl: 'https://api.spotify.com/v1/albums/new123',
+    };
+  
+    const response = await request(app)
+      .post('/api/v1/albums')
+      .set('Cookie', `token=${user.token}`) 
```

Comment:
We should be able to get a cookie the same way you've done in the `should test the getAlbumWithAllUsersWhoPurchasedIt endpoint - Success Case` test